pci/src/class.rs

```diff
@@ -2,16 +2,21 @@ use crate::error;
 use core::convert::TryFrom;
 
 macro_rules! class_enum {
-    (pub enum $name:ident<NoProgIf> {
-        $(
-            // $($m:meta)*
-            $variant:ident = $value:expr
-        ),+
-        $(,)?
-    }) => {
+    (
+        $(#[$m:meta])*
```

Comment:
I couldn't remember if you had to have a `#[...]` around the meta matcher or not (as you can tell) — thanks for fixing this!